charts/cluster-api-runtime-extensions-nutanix/defaultclusterclasses/eks-cluster-class.yaml

```diff
@@ -77,7 +77,7 @@ spec:
       sshKeyName: ""
 ---
 apiVersion: bootstrap.cluster.x-k8s.io/v1beta2
-kind: EKSConfigTemplate
+kind: NodeadmConfigTemplate
```

Comment:
This won't work without without handling it in the handlers right?
I think we should keep it out of this PR and have a follow up to support Kubernetes v1.33

Reply:
Supporting multiple classes based on version is going to be difficult. We can add these CRD changes all at once now, and I'm issuing a follow up PR to change the handlers.

Reply:
Oh yeah totally not suggesting we support both. But making this change in this PR would break the example until the follow up.
We can do keep the ClusterClass and the examples as is and it should continue to work with `EKSConfigTemplate`.